ui/packages/app/src/project_setting/form/Labels.js

```diff
@@ -107,14 +109,15 @@ export const Labels = ({
     });
 
     setLabels(newLabels);
-
-    console.log(isDisabled || items.length === 0);
   };
 
   return (
     <EuiFormRow isInvalid={!isValidLabels} error={labelError}>
       <EuiFlexGroup direction="column" gutterSize="m">
         {items.map((element, idx) => {
+          const isFieldDisabled =
+            isDisabled ||
+            (labelsBlacklist && config.LABELS_BLACKLIST.includes(element.key));
```

Comment:
Please revisit this comment https://github.com/caraml-dev/mlp/pull/99/files#r1627139561 and decide if you still need the variable `labelsBlacklist`.

Also, similar to the comment on O(1) lookups for the API server, can we use a set instead of a list/array? This will involve refactoring `config.LABELS_BLACKLIST` so it's no longer an array but a set.